$OpenBSD: patch-spamd_spamd_raw,v 1.5 2014/04/25 09:20:27 bluhm Exp $

This patch should be removed after OpenBSD got support for AI_ADDRCONFIG.

--- spamd/spamd.raw.orig	Fri Feb  7 09:36:38 2014
+++ spamd/spamd.raw	Thu Apr 24 01:45:37 2014
@@ -59,7 +59,7 @@ BEGIN {
     Socket->VERSION(1.96);  # provides NIx_NOSERV, and Exporter tag :addrinfo
   # Socket->VERSION(1.97);  # IO::Socket::IP depends on Socket 1.97
     import Socket qw(/^(?:AI|NI|NIx|EAI)_/);
-    &AI_ADDRCONFIG; &AI_PASSIVE;  # enable inlining
+    &AI_PASSIVE;  # enable inlining
     &NI_NUMERICHOST, &NI_NUMERICSERV; &NIx_NOSERV; 1;
   };
 
